server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/environment_config_representer.rb

```diff
@@ -0,0 +1,71 @@
+##########################################################################
+# Copyright 2015 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  module Config
+    class EnvironmentConfigRepresenter < ApiV1::BaseRepresenter
+      alias_method :environment, :represented
+
+      error_representer({
+                          'variables'     => 'environment_variables',
+                        })
+
+      link :self do |opts|
+        opts[:url_builder].apiv1_admin_environment_url(name: environment.name)
+      end
+
+      link :doc do |opts|
+        'http://api.go.cd/#environment_config'
+      end
+
+      link :find do |opts|
+        opts[:url_builder].apiv1_admin_environment_url(name: '__environment_name__').gsub(/__environment_name__/, ':environment_name')
+      end
+
+      property :name, case_insensitive_string: true
+
+      delegate :name, :name=, to: :environment
```

Comment:
Please move this delegate to after all `property` and `collection` declarations, to make it easy to follow the code (and the convention)